doc/astspec.txt

```diff
@@ -290,7 +372,7 @@ AST:
 Ranges
```

Comment:
See #2929. Seems like `nnkRange` isn't used anywhere, but `..` is treated like an infix operator?